docs/getting-started.md

```diff
@@ -0,0 +1,105 @@
+# Getting Started - BAG Bevragen API
+De 'BAG Bevragen' API levert actuele gegevens uit de Basisregistratie Adressen en Gebouwen. Dit betekent dat geen historische gegevens kunnen worden gevraagd en dat gegevens met een eindstatus niet worden geleverd.
+
+De API is gespecificeerd met behulp van de OpenAPI Specifications (OAS3).
+
+Om aan te sluiten kun je de volgende stappen doorlopen:
+1. [Meld je aan bij het kadaster om toegang te krijgen](#Aanmelden-om-aan-te-sluiten)
+2. [Bekijk de functionaliteit en specificaties](#Functionaliteit)
+3. [Probeer en test de API](#Probeer-en-test-de-API)
+
+## Aanmelden om aan te sluiten
+Vraag [een API-key aan](https://formulieren.kadaster.nl/aanvragen_bag_api_key_testomgeving) om toegang te krijgen tot de BAG API.
+
+## Functionaliteit
+U kunt de Open API Specificaties (OAS3) van de API bekijken in [Swagger-formaat](https://petstore.swagger.io/?url=https://raw.githubusercontent.com/VNG-Realisatie/Haal-Centraal-BAG-Bevragen/master/api-specificatie/openapi.yaml).
+
+De (resolved) OAS3 is hier te downloaden: [openapi.yaml](../api-specificatie/resolved/openapi.yaml).
+
+### Beschikbare resources
+De API kent de volgende resources:
+- adressen: hierin zijn samenhangende en gerelateerde gegevens samengevoegd uit de nummeraanduiding, openbare ruimte en woonplaats die samen een adres vormen
+- adresseerbareobjecten: dit kan een verblijfsobject, ligplaats of standplaats zijn
+- nummeraanduidingen
+- openbareruimten
+- woonplaatsen
+- panden
+
+Voor de verschillende resources wordt de uri samengesteld met de identificatie (lokaal id) van het onderliggende BAG-object. Dit is een 16 cijferige, of in geval van woonplaats 4 cijferige identificatie. Voor adressen wordt de nummeraanduiding identificatie gebruikt in de uri.  
```

Comment:
Er is veel om te doen geweest, maar de term lokaal id komt in de yaml specificatie nu niet meer voor. Ik denk dat het dan beter is deze term hier ook niet te noemen, dat levert misschien juist weer verwarring op.